modules/fts/pages/fts-custom-filters.adoc

```diff
@@ -4,7 +4,7 @@ Custom filters can be viewed and modified from the index’s configuration page
 
 == Add Custom Filter
 
-In order to add a custom filter to a Full Text Index via the Couchbase Capella UI, the following permissions are required:
+In order to add a custom filter to a Full Text Index via the Couchbase Cloud UI, the following permissions are required:
```

Comment:
Hmm, why are we remove mention of `Capella` here?

Reply:
```suggestion
In order to add a custom filter to a Full Text Index via the Couchbase Capella UI, the following permissions are required:
```